app/controllers/login.js

```diff
@@ -22,6 +22,7 @@ export default Ember.Controller.extend({
         .authenticate('authenticator:conduit', { email, password })
         .then(() => {
           this.transitionToRoute('home');
+          get(this, 'session').authorize('authorizer:conduit');
```

Comment:
From what I understood from simple-auth docs is that this doesn't need to return a promise. 
It just runs the authorizer on the session.